### PR TITLE
Add with credentials to loadObj options

### PR DESCRIPTION
**Pull request by manmaor**
Sometimes requests credentials are required to be successful, in my case it's requests to the AWS CDN. 
So I added `withCredentials` to loadObj options, so requests will have credentials.

options example:
```
  const windTurbineOptions = {
    obj: awsCdnUrl,
    type: 'gltf',
    scale: 1,
    units: 'meters',
    anchor: 'center',
    withCredentials: true,
  };
  tb.loadObj(windTurbineOptions, (model) => {
    const { longitude, latitude } = location
    model.setCoords([longitude, latitude, groundAlt]);
    tb.add(model);
  });
```

### CODE REVIEW

**Comment by jscastro76** [comment]
This PR includes also a change in the vertex shader for buildings shadow

**Comment by manmaor** [comment]
@jscastro76 It seems that the build command generated these changes, Would you prefer That I will only upload the code changes?

**Comment by jscastro76** [comment]
yes please, the credentials part wasn't an identified scenario, but I'm open to consider it.
The vertex shader is something that requires a lot of tests before approval

**Comment by manmaor** [comment]
@jscastro76 Done 😄

**Comment by manmaor** [comment]
@jscastro76 Hi, Is there anything else that needs to be done to get this approved?

**Review by jscastro76 — APPROVED** [review]

**Comment by jscastro76** [comment]
Approved! Thanks for your contribution!

**Pull request merged**
